docs/running/cloud/amazon.rst

```diff
@@ -118,6 +118,10 @@ To further break down each of these commands:
 
     **--keyPairName id_rsa** --- The name of your key pair, which should be "id_rsa" if you've followed this tutorial.
 
+.. note::
+
+   You can set the TOIL_AWS_TAGS with a JSON dictionary to specify arbitrary tags for AWS resources
```

Comment:
"the TOIL_AWS_TAGS" sounds pretty weird, as does "set ... with". And JSON doesn't call them "dictionaries" but instead "objects". And it's not explicitly stated here that the format is going to be tag name to tag value. And an example would be nice. Also the variable name should maybe be formatted as code.

```suggestion
   You can set the ``TOIL_AWS_TAGS`` environment variable to a JSON object to specify arbitrary tags for AWS resources. For example, if you ``export TOIL_AWS_TAGS='{"project-name": "variant-calling"}'`` in your shell before using Toil, AWS resources created by Toil will be tagged with a ``project-name`` tag with the value ``variant-calling``.
```